cmd/amppkg/main.go

```diff
@@ -120,8 +121,15 @@ func main() {
 	defer rtvCache.StopCron()
 
 	var overrideBaseURL *url.URL
-	if *flagDevelopment {
-		overrideBaseURL, err = url.Parse(fmt.Sprintf("https://localhost:%d/", config.Port))
+	if *flagStaging == "" {
+		if *flagDevelopment {
+			overrideBaseURL, err = url.Parse(fmt.Sprintf("https://localhost:%d/", config.Port))
+			if err != nil {
+				die(errors.Wrap(err, "parsing development base URL"))
+			}
+		}
+	} else {
```

Comment:
I think this would read a bit more easily as:

```
if *flagStaging != "" {
  ...
} else if *flagDevelopment {
  ...
}
```

Reply:
Good suggestion, done.